### PR TITLE
Update files with PEP8 standards in folder dojo/tools #004

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/tools/

Folder affected
dojo/tools/hydra
dojo/tools/huskyci
dojo/tools/ibm_app
dojo/tools/immuniweb
dojo/tools/intsights
dojo/tools/jfrog_xray_api_summary_artifact
dojo/tools/jfrog_xray_unified
dojo/tools/jfrogxray
dojo/tools/kics
dojo/tools/kiuwan
dojo/tools/meterian
dojo/tools/microfocus_webinspect
dojo/tools/mobsfscan

### CODE REVIEW

File: dojo/tools/huskyci/parser.py [hunk 1]
```diff
@@ -31,29 +30,30 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/huskyci/parser.py:34]:
```suggestion
            except Exception:
```

File: dojo/tools/huskyci/parser.py [hunk 2]
```diff
@@ -31,29 +30,30 @@ def parse_json(self, json_output):
...
                 tree = json.loads(data)
-        except:
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/huskyci/parser.py:36]:
```suggestion
            except Exception:
```

File: dojo/tools/kiuwan/parser.py [hunk 3]
```diff
@@ -50,33 +52,55 @@ def get_findings(self, filename, test):
...
             finding.references = "Not provided!"
             finding.mitigation = "Not provided!"
-            finding.severity = findingdict['severity']
+            finding.severity = findingdict["severity"]
             finding.static_finding = True
             try:
-                finding.cwe = int(row['CWE'])
-            except:
+                finding.cwe = int(row["CWE"])
+            except BaseException:
```
Comment (Maffooch) [inline dojo/tools/kiuwan/parser.py:103]:
```suggestion
            except Exception:
```

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/tools/kiuwan/parser.py (hunk 3 above)
Comment (ajtortolero) [reply dojo/tools/kiuwan/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8304/commits/3683282176e99b2e46e9ecd1bd825090dcab1191

File: dojo/tools/huskyci/parser.py (hunk 2 above)
Comment (ajtortolero) [reply dojo/tools/huskyci/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8304/commits/3683282176e99b2e46e9ecd1bd825090dcab1191

File: dojo/tools/huskyci/parser.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/tools/huskyci/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8304/commits/3683282176e99b2e46e9ecd1bd825090dcab1191

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by devGregA — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Pull request merged**